src/NetCoreInstaller/Targets/packaging/windows/NetCoreInstaller/netcoreinstaller.wxs

```diff
@@ -0,0 +1,494 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi"
+     xmlns:bal="http://schemas.microsoft.com/wix/BalExtension"
+     xmlns:util="http://schemas.microsoft.com/wix/UtilExtension"
+     xmlns:dnc="http://schemas.microsoft.com/wix/DotNetCoreBootstrapperExtension"
+     xmlns:dep="http://schemas.microsoft.com/wix/DependencyExtension">
+    <?include "variables.wxi" ?>
+
+    <Bundle Name="$(var.InstallerName)" Version="$(var.InstallerVersion)" Manufacturer="$(var.Manufacturer)" UpgradeCode="$(var.UpgradeAndProviderCode)"
+            dep:ProviderKey="$(var.UpgradeAndProviderCode)" AboutUrl="https://dotnet.github.io/" Compressed="yes">
+        <BootstrapperApplicationRef Id="ManagedBootstrapperApplicationHost">
+            <dnc:DotNetCoreBootstrapperApplication ShowCopyright="yes" ShowTelemetryCheckBox="yes" ShowLicenseCheckBox="yes"
```

Comment:
You can remove the ShowCopyright attribute since the bootstrapper footer is disabled by default

Reply:
Done